python/ansible_select/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           select 1.0
+
+name                ansible_select
+version             0.1
+categories          sysutils
```

Comment:
There's an error in the build, try changing to categories-append.

Reply:
Changed...

... but the Travis failure looked like it was a general timeout. Not sure if it's linked.

We'll see when the updated PR build finishes, below, I guess...!